patches/minecraft/net/minecraft/client/gui/inventory/GuiContainer.java.patch

```diff
@@ -128,11 +161,19 @@
                  {
                      this.func_184098_a(this.field_147006_u, this.field_147006_u.field_75222_d, i, ClickType.SWAP);
                      return true;
-@@ -686,4 +691,16 @@
+@@ -686,4 +697,24 @@
              this.field_146297_k.field_71439_g.func_71053_j();
          }
      }
++    
++    @Override
++    protected void func_146284_a(net.minecraft.client.gui.GuiButton button) throws IOException {
++    	this.guiTabs.actionPerformed(button);
++    }
 +
++    public java.util.List<net.minecraftforge.client.player.inventory.tabs.GuiTab> getTabs(){
```

Comment:
No idea. might be. I am not a forge developper or anything, so I thought i'd keep it outside for now